src/index.js

```diff
@@ -1 +1,68 @@
 import './styles.css';
+import { NewApiGame, GetScores } from '../module/api.js';
+
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+
+const getAllScores = async () => {
+  const allScores = await GetScores();
+
+  allScores.result.sort((a, b) => b.score - a.score);
+
+  allScores.result.forEach((item) => {
+    const scoreList = document.getElementById('score-list');
+    const scoreListItem = document.createElement('li');
+    scoreListItem.className = 'list-item';
+    scoreListItem.innerHTML = `${item.user}:  ${item.score}`;
+    scoreList.appendChild(scoreListItem);
+  });
+  const scoreLength = allScores.result.length;
+  const totalScores = document.querySelector('small');
+  totalScores.className = 'score-length';
+  totalScores.innerText = ` Total Number of Users: ${scoreLength}`;
+};
+
+// message display for successful adding
+// const dispMessage = () => {
+//   messageElt.innerText = 'Score added successfully, click "refresh" to display';
+//   messageElt.className = 'small'
```

Comment:
- [OPTIONAL] Please kindly delete the comment lines of code to have a clean codebase 🙃 🙃